packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -713,3 +821,9 @@ function makeSendValuesBaseOptionsFromLegacy(
     ? { none: true }
     : { all: true };
 }
+
+export function computeExecutableSchemaId(schema: string | GraphQLSchema): string {
+  let schemaDocument = isString(schema) ? schema
+    : stripIgnoredCharacters(printSchema(lexicographicSortSchema(schema)));
```

Comment:
Didn't we say we want to remove `lexicographicSortSchema`?

Reply:
No. If there was some last minute discussion yesterday that was ephemeral in Slack I think we resolved by saying we've documented the decision and will not change it in the last minute for the soft launch.